web/src/services/api/sceneApi.ts

```diff
@@ -43,8 +43,9 @@ export default () => {
             property: data.node.property,
             rootLayerId: data.node.rootLayerId,
             newLayers: data.node.newLayers,
-            tags: data.node.tags,
             stories: data.node.stories,
+            styles: data.node.styles,
```

Comment:
Why do we call them styles on the scene? Shouldn't we keep the domain naming consistent and call this `appearances`?